### PR TITLE
fix: UI issue on marketplace filters

### PR DESCRIPTION
**Pull request by csuadev**
## Proposed changes (including videos or screenshots)
Current:
image

image

Fixed:
image

image

## Issue(s)
[https://rocketchat.atlassian.net/browse/AECO-352](https://rocketchat.atlassian.net/browse/AECO-352)

### CODE REVIEW

File: .changeset/cyan-mangos-do.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+chore: fix UI issue on marketplace filters
```
Comment (juliajforesti) [inline .changeset/cyan-mangos-do.md:5]:
```suggestion
fix: UI issue on marketplace filters
```

**Review by aleksandernsilva — DISMISSED** [review]

File: apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx [hunk 2]
```diff
@@ -15,33 +16,38 @@ const CategoryDropDownAnchor = forwardRef<HTMLElement, CategoryDropDownAnchorPro
...
+			pb={9}
+			pie={7}
+			pis={14}
 			{...props}
 		>
 			{selectedCategoriesCount > 0 && (
 				<Box
-					mie={6}
-					borderRadius='x32'
+					is={Button}
```
Comment (juliajforesti) [inline apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx:38]: why is this a `Button`? It's not semantically correct to have a **button inside a button**

We do have a similar component: `Badge` that I would suggest applying here. The design is not exactly the same, but we should avoid having **slightly** different and customized components all around...

**Review by juliajforesti — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx [hunk 3]
```diff
@@ -15,33 +16,40 @@ const CategoryDropDownAnchor = forwardRef<HTMLElement, CategoryDropDownAnchorPro
...
 			borderColor={selectedCategoriesCount ? 'none' : 'light'}
+			borderRadius='x4'
 			borderWidth={selectedCategoriesCount ? 'none' : 'x1'}
-			{...(selectedCategoriesCount ? { primary: true } : { bg: 'surface-light' })}
+			display='flex'
+			flexGrow={1}
+			flexShrink={1}
+			justifyContent='space-between'
+			minWidth='x144'
+			pb={9}
```
Comment (juliajforesti) [inline apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx:32]:
```suggestion
			height='x40'
```

File: apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx [hunk 4]
```diff
@@ -15,33 +16,40 @@ const CategoryDropDownAnchor = forwardRef<HTMLElement, CategoryDropDownAnchorPro
...
+			pie={7}
+			pis={14}
```
Comment (juliajforesti) [inline apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx:34]:
```suggestion
			pis={14}
			lineHeight='unset'
			rcx-input-box
```

File: apps/meteor/client/views/marketplace/components/RadioDropDown/RadioDownAnchor.tsx [hunk 5]
```diff
@@ -14,22 +15,24 @@ const RadioDownAnchor = forwardRef<HTMLElement, RadioDropdownAnchorProps>(functi
...
-			h='x40'
+			display='flex'
 			flexGrow={1}
 			flexShrink={1}
+			fontScale='p2'
+			h='x40'
+			justifyContent='space-between'
+			minWidth='x144'
+			pie={10}
+			pis={14}
```
Comment (juliajforesti) [inline apps/meteor/client/views/marketplace/components/RadioDropDown/RadioDownAnchor.tsx:35]:
```suggestion
			pis={14}
			rcx-input-box
```

File: apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx [hunk 6]
```diff
@@ -1,4 +1,5 @@
 import { Box, Button, Icon } from '@rocket.chat/fuselage';
```
Comment (juliajforesti) [inline apps/meteor/client/views/marketplace/components/CategoryFilter/CategoryDropDownAnchor.tsx:1]:
```suggestion
import type { Button } from '@rocket.chat/fuselage';
import { Box, Icon } from '@rocket.chat/fuselage';
```

**Review by juliajforesti — APPROVED** [review]

**Pull request merged**